packages/Discretization/src/DTK_PointSearch_def.hpp

```diff
@@ -81,6 +84,90 @@ void buildTopo( Kokkos::View<int *, DeviceType> imported_cell_indices,
     DTK_REQUIRE( ArborX::lastElement( topo_size_sum ) == n_imports );
 #endif
 }
+
+template <typename ViewType>
+void sendDataAcrossNetwork( ArborX::Details::Distributor const &distributor,
+                            std::pair<ViewType, ViewType> data )
+{
+    ArborX::Details::DistributedSearchTreeImpl<
+        typename ViewType::device_type>::sendAcrossNetwork( distributor,
+                                                            data.first,
+                                                            data.second );
+}
+
+template <typename T, typename... Targs>
+void sendDataAcrossNetwork( ArborX::Details::Distributor const &distributor,
+                            std::pair<T, T> d, Targs... data )
+{
+    sendDataAcrossNetwork( distributor, d );
+    sendDataAcrossNetwork( distributor, data... );
+}
```

Comment:
CC'ing @mastereinlad so he sees the two `sendDataAcrossNetwork` overloads above.